wagtail/admin/views/mixins.py

```diff
@@ -10,47 +10,7 @@
 from django.utils.translation import gettext as _
 from xlsxwriter.workbook import Workbook
 
-from wagtail.admin.forms.search import SearchForm
 from wagtail.coreutils import multigetattr
-from wagtail.search.backends import get_search_backend
-from wagtail.search.index import class_is_indexed
-
-
-class SearchableListMixin:
```

Comment:
🤔 this feels like a useful mixin that allows, say, package developers to use without the need `IndexView`. Perhaps we'll want to take stock of the various admin views and interfaces and their use all around?

Note: that this is very unscientific at this point

Reply:
Yeah, I initially didn't remove it but I didn't see it being used anywhere else in Wagtail. I guess we can leave it as it is. I haven't searched whether people use it, though.

Reply:
This code is dead, undocumented, and probably now incompatible with Wagtail's own ``IndexView``. I'm in favour of removing it, the search API should be simple enough to integrate into any custom views.